viewing_party/party.py

```diff
@@ -1,23 +1,196 @@
 # ------------- WAVE 1 --------------------
 
+from pickle import NONE
+import copy
+
 def create_movie(title, genre, rating):
-    pass
+    if not (title and genre and rating): # if title or genre or rationg is None return None
+        return None
+    
+    new_movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    
+    for watched in user_data:
+        user_data[watched] = [movie]     
+    
+    return user_data

```

Comment:
Careful here! We don't want to accidentally overwrite the user_data's "watched" list that's already been supplied to us in the tests in test_wave_01.py. It did pass the tests, but unfortunately the tests did not account for "watched" lists with more than one movie in it. 

Another thing to consider: since we know the key we want to look into already, let's not use the for loop. There might be only one key in this particular dictionary, but we can't always assume. So let's go right to user_data["watched"]

```suggestion
    user_data["watched"].append(movie)
    
    return user_data
```